### ISSUE TITLE
English docs and comments

### ISSUE THREAD

**Issue by NikiSchlifke** [haiwen/seafile-ansible#1]
This seems to be an interesting setup, unfortunately I do not understand Chinese.
Any chance this can be added?

**Comment by DerDanilo** [comment]
Yes, it would be awesome if the comments could be done in English generally. Way more people could then understand and hence use code from the Seafile devs to deploy their code. :)
